src/cosmosdb-preview/azext_cosmosdb_preview/_help.py

```diff
@@ -7,6 +7,154 @@
 
 from knack.help_files import helps  # pylint: disable=unused-import
 
+helps['cosmosdb mongocluster firewall rule'] = """
+type: group
+short-summary: CosmosDB for MongoDB cluster firewall rule.
+"""
+
+helps['cosmosdb mongocluster firewall rule create'] = """
+type: command
+short-summary: Create a CosmosDB for MongoDB cluster firewall rule.
+examples:
+  - name: Create a CosmosDB for MongoDB cluster firewall rule.
+    text: |
+      az cosmosdb mongocluster firewall rule create \\
+        --cluster-name MyCluster \\
+        --resource-group MyResourceGroup \\
+        --rule-name MyRule \\
+        --start_ip_address MyStartIpAddress \\
+        --end_ip_address MyEndIpAddress \\
+"""
+
+helps['cosmosdb mongocluster firewall rule update'] = """
+type: command
+short-summary: Create a CosmosDB for MongoDB cluster firewall rule.
+examples:
+  - name: Create a CosmosDB for MongoDB cluster firewall rule.
+    text: |
+      az cosmosdb mongocluster firewall rule create \\
+        --cluster-name MyCluster \\
+        --resource-group MyResourceGroup \\
+        --rule-name MyRule \\
+        --start_ip_address MyStartIpAddress \\
+        --end_ip_address MyEndIpAddress \\
+"""
+
+helps['cosmosdb mongocluster firewall rule list'] = """
+type: command
+short-summary: Lists firewall rule on a mongo cluster.
+examples:
+  - name: Lists a Mongo Cluster firewall rule in a resource group.
+    text: |
+      az cosmosdb mongocluster list --cluster-name MyCluster --resource-group MyResourceGroup
+"""
+
+helps['cosmosdb mongocluster firewall rule show'] = """
+type: command
+short-summary: Get a Mongo Cluster firewall rule.
+examples:
+  - name: Gets a Mongo Cluster firewall rule. If the firewall rule doesnot exist a NotFound response is returned.
+    text: |
+      az cosmosdb mongocluster show --cluster-name MyCluster --resource-group MyResourceGroup --rule-name MyRuleName
+"""
+
+helps['cosmosdb mongocluster firewall rule delete'] = """
+type: command
+short-summary: Delete a Mongo Cluster firewall rule.
+examples:
+  - name: Delete a Mongo Cluster firewall rule. If the firewall rule does not exist a NotFound response is returned.
```

Comment:
NoContent is returned if the resource does not exist.

Reply:
Tests should reflect this.

Reply:
Added tests